File: abc/abc192/a.java
// https://atcoder.jp/contests/abc192/submissions/41451735

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int x = stdIn.nextInt();
    System.out.println(100 - (x % 100));
  }
}

File: abc/abc192/a.cs
// https://atcoder.jp/contests/abc192/submissions/46503292

using System;

public class A
{
  static void Main()
  {
    int x = int.Parse(Console.ReadLine());
    if(x % 100 == 0)
    {
      Console.Write(100);
    }
    else
    {
      Console.Write(100 - x % 100);
    }
  }
}

